src/preferences/dialog/dlgpreflv2.cpp

```diff
@@ -111,6 +111,14 @@ void DlgPrefLV2::slotDisplayParameters() {
     lv2_vertical_layout_params->addStretch();
 }
 
+void DlgPrefLV2::slotUpdate() {
+    // TODO: Implement this
+}
+
+void DlgPrefLV2::slotResetToDefaults() {
+    // TODO: Implement this
+}
+
```

Comment:
I have removed DlgPrefLV2 in #2618, so there is nothing to do here.

Reply:
I'll add a comment